packages/test/test-drivers/src/routerliciousTestDriver.ts

```diff
@@ -57,7 +59,7 @@ function getEndpointConfigFromEnv(r11sEndpointName: string) {
     }
     assert(configStr, `Missing config for ${r11sEndpointName}`);
     const config = JSON.parse(configStr);
-    return getConfig(config.host, config.tenantId, config.tenantSecret);
+    return getConfig(config.host, config.tenantId, config.tenantSecret, config.useWholeSummary);
```

Comment:
>useWholeSummary [](http://example.com/codeflow?start=79&length=15)

should we just allow the whole IRouterliciousDriverPolicies here?

Reply:
How do we do that? Stringified JSON?